xml/admin_monitoring_alerting.xml

```diff
@@ -110,41 +110,87 @@
+    the nature of how Grafana is implemented in Ceph Dashboard, this means that
+    two working connections are required in order to be able to see Grafana
+    graphs in Ceph Dashboard:
+  </para>
   <itemizedlist>
     <listitem>
       <para>
-        <emphasis role="bold"><literal>ssl_cert</literal></emphasis>
+        The backend (Ceph Mgr module) needs to verify the existence of the
+        requested graph. If this request succeeds, it lets the frontend know
+        that it can safely access Grafana.
       </para>
     </listitem>
     <listitem>
       <para>
-        <emphasis role="bold"><literal>ssl_key</literal></emphasis>
+        The frontend then requests the Grafana graphs directly from the user's
+        browser using an iframe. The Grafana instance is accessed directly
+        without any detour through Ceph Dashboard.
       </para>
     </listitem>
   </itemizedlist>
   <para>
-    The &dashboard; embeds the &grafana; dashboards via HTML <literal>iframe</literal>
-    elements. If &grafana; is configured without SSL/TLS support or if SSL
-    uses self-signed certificates, most browsers will block the embedding of
-    insecure content into a secured web page if the SSL support in the
-    dashboard has been enabled (which is the default configuration). If you
-    can not see the embedded &grafana; dashboards in &dashboard;,
-    check your browser's documentation on how to unblock mixed content
-    or how to accept self-signed certificates. Alternatively, consider
-    enabling SSL/TLS support in &grafana;, using a certificate that is
-    issued by a certificate authority (CA) known to the browser.
+    Now, it might be the case that your environment makes it difficult for the
+    user's browser to directly access the URL configured in Ceph Dashboard. To
+    solve this issue, a separate URL can be configured which will solely be used
+    to tell the frontend (the user's browser) which URL it should use to access
+    Grafana.
   </para>
-  <para>For more information on supplying your own SSL certificates,
-    see <xref linkend="cert-sign-CA"/>, for generating a self-signed or trusted
-    third-party certificate using OpenSSL, see <xref linkend="self-sign-certificates-openssl"/>,
-    for creating your own CA signed certificate, see <xref linkend="self-sign-certificates"/>,
-    and for creating your own custom CA signed certificate, see
-    <xref linkend="cert-sign-custom-CA"/>.
+  <para>
+    To change the URL that is returned to the frontend issue the following command:
   </para>
+<screen>&prompt.cephuser;ceph dashboard set-grafana-frontend-api-url <replaceable>GRAFANA-SERVER-URL</replaceable></screen>
+  <para>
+    If no value is set for that option, it will simply fall back to the value of
+    the <replaceable>GRAFANA_API_URL</replaceable> option, which is set
+    automatically and periodically updated cephadm. If set, it will instruct the
```

Comment:
```suggestion
    automatically and periodically updated by &cephadm;. If set, it will instruct the
```

Reply:
That was a leftover from the copy of the SES7 docs update. I removed the period update and replaced cephadm with deepsea.